### DISCUSSION TITLE
No subjects found

### DISCUSSION THREAD

**Discussion by liujiynagchian** [QSMxT/QSMxT#130] [Q&A]
Hello everyone. I have converted my Philips dicom data to Nifti data outside of the program using MRIcrogl. Within the qsmxt software, I then converted my data to be BIDS-compliant:

bids/
├── sub-1
│   └── anat
│       ├── sub-1_echo-1_part-mag_MEGRE.json
│       ├── sub-1_echo-1_part-mag_MEGRE.nii
│       ├── sub-1_echo-1_part-phase_MEGRE.json
│       ├── sub-1_echo-1_part-phase_MEGRE.nii
│       ├── sub-1_echo-2_part-mag_MEGRE.json
│       ├── sub-1_echo-2_part-mag_MEGRE.nii
│       ├── sub-1_echo-2_part-phase_MEGRE.json
│       └── sub-1_echo-2_part-phase_MEGRE.nii

However, when I enter the qsmxt , I receive the following message:

(command:   qsmxt /Users/liujiyang/Desktop/bids)

[INFO]: QSMxT v7.1.0
[INFO]: Python interpreter: /opt/miniconda-4.7.12.1/bin/python3.8
[INFO]: Command: qsmxt /Users/liujiyang/Desktop/bids --labels_file '/opt/miniconda-4.7.12.1/lib/python3.8/site-packages/qsmxt/aseg_labels.csv' --auto_yes
[INFO]: Guidelines compliant! (see https://doi.org/10.1002/mrm.30006)
[INFO]: Available memory: 6.412 GB
[ERROR]: No subjects found in /Users/liujiyang/Desktop/bids/sub*
[INFO]: Errors occurred!
[INFO]: Finished

**Comment by astewartau** [comment]
Hi @liujiynagchian, thanks for reaching out! The no subjects message is strange, given that your BIDS directory looks okay... The `tree` output does look good. To confirm, can you also paste the output of `ls /Users/liujiyang/Desktop/bids/sub-1/anat/*phase*` and `ls /Users/liujiyang/Desktop/bids/sub-1/anat/*mag*` to confirm that these full patterns definitely generate matches? 

The `dicom-convert` tool in QSMxT should be able to convert Philips data to BIDS as well if you have valid DICOMs.

**Reply by liujiynagchian** [reply]
I run a Docker container in Mac to use qsmxt, the command is as follows：
docker run -it  --platform linux/amd64 -v /Users/liujiyang/Desktop/bids:/neurodesktop-storage vnmd/qsmxt_7.1.0:20240809


(base) root@b25cbd0ede3d:/opt# ls /Users/liujiyang/Desktop/bids/sub-1/anat/*phase*
ls: cannot access '/Users/liujiyang/Desktop/bids/sub-1/anat/*phase*': No such file or directory

(base) root@b25cbd0ede3d:/opt# ls /Users/liujiyang/Desktop/bids/sub-1/anat/*mag*
ls: cannot access '/Users/liujiyang/Desktop/bids/sub-1/anat/*mag*': No such file or directory
(base) root@b25cbd0ede3d:/opt#

**Reply by astewartau** [reply]
Ah, I believe this is a Docker issue. When you mount the user directory to /neurodesktop-storage in the container, then that is where all the data will be. 

The easiest way to address this is to create a /neurodesktop-storage folder on your computer, place any necessary data there, and run exactly the Docker command from the QSMxT website, which will mount that directory to the same location within the container.

If these instructions are unclear or you have more questions, please let me know. 😊
